src/constants/pinning.js

```diff
@@ -32,6 +32,12 @@ const pinningServiceTemplates = [
     apiEndpoint: 'https://api.filebase.io/v1/ipfs',
     visitServiceUrl: 'https://docs.filebase.com/api-documentation/ipfs-pinning-service-api'
   },
+  {
+    name: 'Functionland',
+    icon: 'https://dweb.link/ipfs/bafybeidgnnvgm6i3pfzhjakgzcdwdbvwtmkkx7vzeiyvtk2b4oxis53vhm?filename=functionland.png',
```

Comment:
Just realized IPFS version is 5000x5000 pixels in size for some reason.
@ehsan6sha  do you have SVG by any chance? If not, we should switch to 512x512.

Reply:
Provisionally, replaced 192k PNG with 7k SVG:
```suggestion
    icon: 'https://dweb.link/ipfs/QmWYEmdYq9Ry2xtb69oZSPXb8Aos24kWdVecsT3txVe38E?filename=functionland.svg',
```

Let's see how it looks.